lib/radius.js

```diff
@@ -494,7 +494,7 @@ Radius.decode_attributes = function(data, attr_hash, vendor, raw_attrs) {
       case "time":
       case "integer":
         if (attr_info[ATTR_MODIFIERS]["has_tag"]) {
-          var buf = new Buffer([0, 0, 0, 0]);
+          var buf = Buffer.alloc([0, 0, 0, 0]);
```

Comment:
Some of these should probably be Buffer.from instead.  I haven't gone over the full list but I don't think a straight replacement with Buffer.alloc will work.  Could you go through the list and verify?

Reply:
Sure thing. I'll revisit this again :)